stdlib/universe/universe.flux

```diff
@@ -98,6 +98,11 @@ builtin logarithmicBins
 builtin sleep // sleep is the identity function with the side effect of delaying execution by a specified duration
 builtin die // die returns a fatal error from within a flux script
 
+// Time weighted average where values at the beginning and end of the range are linearly interpolated.
```

Comment:
Was there an explicit decision to put this in universe? I'm starting to be worried that we might be polluting the namespace a lot now.

Reply:
No explicit decision, I just wasn't sure what package this function should belong to. I agree about polluting the universe package though. I'm open to any suggestions.